DXMainClient/DXGUI/Generic/MainMenu.cs

```diff
@@ -538,9 +538,12 @@ public void PostInit()
                     lblUpdateStatus.DrawUnderline = false;
                 }
                 else if (UserINISettings.Instance.CheckForUpdates)
+                {
                     CheckForUpdates();
-                else
+                }
+                {
                     lblUpdateStatus.Text = "Click to check for updates.".L10N("UI:Main:ClickToCheckUpdate");
+                }
```

Comment:
?

Reply:
You said 'inconsistent braces', I added braces to the entire if-elseif-else block.

Reply:
![image](https://user-images.githubusercontent.com/17500545/169694409-30da5be3-a1d5-4ec5-9c99-adf214f06961.png)

Reply:
Yeah okay must have accidentally wiped the `else` out. Weird that it generates no build errors.

Reply:
Probably because operator braces can be used without anything else both in C# and C++.